### ISSUE TITLE
Can not update - windows version

### ISSUE THREAD

**Issue by mirek190** [runew0lf/RuinedFooocus#136]
LOG:

E:\DIFFUSION\RuinedFooocus_win64_1-25-2>.\python_embeded\python.exe -s RuinedFooocus\entry_with_update.py
You have the latest version
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
RuinedFooocus version: 1.37.1
Comfy Backend update check complete.
Installing requirements
Traceback (most recent call last):
  File "E:\DIFFUSION\RuinedFooocus_win64_1-25-2\RuinedFooocus\entry_with_update.py", line 49, in <module>
    from launch import *
  File "E:\DIFFUSION\RuinedFooocus_win64_1-25-2\RuinedFooocus\launch.py", line 238, in <module>
    prepare_environment()
  File "E:\DIFFUSION\RuinedFooocus_win64_1-25-2\RuinedFooocus\launch.py", line 101, in prepare_environment
    run_pip(f'install -r "{requirements_file}"', "requirements")
  File "E:\DIFFUSION\RuinedFooocus_win64_1-25-2\RuinedFooocus\modules\launch_util.py", line 98, in run_pip
    return run(
  File "E:\DIFFUSION\RuinedFooocus_win64_1-25-2\RuinedFooocus\modules\launch_util.py", line 91, in run
    raise RuntimeError("\n".join(error_bits))
RuntimeError: Couldn't install requirements.
Command: "E:\DIFFUSION\RuinedFooocus_win64_1-25-2\python_embeded\python.exe" -m pip install -r "requirements_versions.txt" --prefer-binary
Error code: 1
stdout: Requirement already satisfied: torchsde==0.2.5 in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 1)) (0.2.5)
Collecting einops==0.4.1 (from -r requirements_versions.txt (line 2))
  Using cached einops-0.4.1-py3-none-any.whl.metadata (10 kB)
Collecting transformers==4.30.2 (from -r requirements_versions.txt (line 3))
  Using cached transformers-4.30.2-py3-none-any.whl.metadata (113 kB)
Collecting safetensors==0.3.1 (from -r requirements_versions.txt (line 4))
  Using cached safetensors-0.3.1-cp310-cp310-win_amd64.whl.metadata (4.6 kB)
Requirement already satisfied: accelerate==0.21.0 in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 5)) (0.21.0)
Collecting pyyaml==6.0 (from -r requirements_versions.txt (line 6))
  Using cached PyYAML-6.0-cp310-cp310-win_amd64.whl.metadata (2.0 kB)
Collecting Pillow==9.2.0 (from -r requirements_versions.txt (line 7))
  Using cached Pillow-9.2.0-cp310-cp310-win_amd64.whl.metadata (9.0 kB)
Collecting scipy==1.9.3 (from -r requirements_versions.txt (line 8))
  Using cached scipy-1.9.3-cp310-cp310-win_amd64.whl.metadata (58 kB)
Collecting tqdm==4.64.1 (from -r requirements_versions.txt (line 9))
  Using cached tqdm-4.64.1-py2.py3-none-any.whl.metadata (57 kB)
Collecting psutil==5.9.5 (from -r requirements_versions.txt (line 10))
  Using cached psutil-5.9.5-cp36-abi3-win_amd64.whl.metadata (21 kB)
Collecting numpy==1.23.5 (from -r requirements_versions.txt (line 11))
  Using cached numpy-1.23.5-cp310-cp310-win_amd64.whl.metadata (2.3 kB)
Requirement already satisfied: pytorch_lightning==1.9.4 in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 12)) (1.9.4)
Collecting omegaconf==2.2.3 (from -r requirements_versions.txt (line 13))
  Using cached omegaconf-2.2.3-py3-none-any.whl.metadata (3.9 kB)
Collecting gradio==3.47.1 (from -r requirements_versions.txt (line 14))
  Using cached gradio-3.47.1-py3-none-any.whl.metadata (17 kB)
Requirement already satisfied: pygit2==1.12.2 in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 15)) (1.12.2)
Requirement already satisfied: aiohttp==3.8.5 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from -r requirements_versions.txt (line 16)) (3.8.5)
Requirement already satisfied: playsound2==1.3.0 in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 17)) (1.3.0)
Requirement already satisfied: diffusers==0.23.0 in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 18)) (0.23.0)
Requirement already satisfied: ftfy==6.1.1 in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 19)) (6.1.1)
Requirement already satisfied: clip_interrogator==0.6.0 in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 20)) (0.6.0)
Collecting opencv-python==4.8.1.78 (from -r requirements_versions.txt (line 21))
  Using cached opencv_python-4.8.1.78-cp37-abi3-win_amd64.whl.metadata (20 kB)
Collecting kornia==0.7.2 (from -r requirements_versions.txt (line 22))
  Using cached kornia-0.7.2-py2.py3-none-any.whl.metadata (12 kB)
Requirement already satisfied: boltons>=20.2.1 in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from torchsde==0.2.5->-r requirements_versions.txt (line 1)) (23.0.0)
Requirement already satisfied: torch>=1.6.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from torchsde==0.2.5->-r requirements_versions.txt (line 1)) (2.1.2+cu118)
Requirement already satisfied: trampoline>=0.1.2 in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from torchsde==0.2.5->-r requirements_versions.txt (line 1)) (0.1.2)
Requirement already satisfied: filelock in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from transformers==4.30.2->-r requirements_versions.txt (line 3)) (3.12.2)
Requirement already satisfied: huggingface-hub<1.0,>=0.14.1 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from transformers==4.30.2->-r requirements_versions.txt (line 3)) (0.16.4)
Requirement already satisfied: packaging>=20.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from transformers==4.30.2->-r requirements_versions.txt (line 3)) (23.1)
Requirement already satisfied: regex!=2019.12.17 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from transformers==4.30.2->-r requirements_versions.txt (line 3)) (2023.10.3)
Requirement already satisfied: requests in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from transformers==4.30.2->-r requirements_versions.txt (line 3)) (2.31.0)
Collecting tokenizers!=0.11.3,<0.14,>=0.11.1 (from transformers==4.30.2->-r requirements_versions.txt (line 3))
  Using cached tokenizers-0.13.3-cp310-cp310-win_amd64.whl.metadata (6.9 kB)
Requirement already satisfied: colorama in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from tqdm==4.64.1->-r requirements_versions.txt (line 9)) (0.4.6)
Requirement already satisfied: fsspec>2021.06.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from fsspec[http]>2021.06.0->pytorch_lightning==1.9.4->-r requirements_versions.txt (line 12)) (2023.6.0)
Requirement already satisfied: torchmetrics>=0.7.0 in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from pytorch_lightning==1.9.4->-r requirements_versions.txt (line 12)) (1.0.3)
Requirement already satisfied: typing-extensions>=4.0.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from pytorch_lightning==1.9.4->-r requirements_versions.txt (line 12)) (4.7.1)
Requirement already satisfied: lightning-utilities>=0.6.0.post0 in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from pytorch_lightning==1.9.4->-r requirements_versions.txt (line 12)) (0.9.0)
Collecting antlr4-python3-runtime==4.9.* (from omegaconf==2.2.3->-r requirements_versions.txt (line 13))
  Using cached antlr4_python3_runtime-4.9.3-py3-none-any.whl
Requirement already satisfied: aiofiles<24.0,>=22.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (23.1.0)
Requirement already satisfied: altair<6.0,>=4.2.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (5.0.1)
Requirement already satisfied: fastapi in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (0.100.1)
Requirement already satisfied: ffmpy in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (0.3.1)
Collecting gradio-client==0.6.0 (from gradio==3.47.1->-r requirements_versions.txt (line 14))
  Using cached gradio_client-0.6.0-py3-none-any.whl.metadata (7.1 kB)
Requirement already satisfied: httpx in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (0.24.1)
Requirement already satisfied: importlib-resources<7.0,>=1.3 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (6.1.1)
Requirement already satisfied: jinja2<4.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (3.1.2)
Requirement already satisfied: markupsafe~=2.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (2.1.3)
Requirement already satisfied: matplotlib~=3.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (3.7.2)
Requirement already satisfied: orjson~=3.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (3.9.2)
Requirement already satisfied: pandas<3.0,>=1.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (1.5.3)
Requirement already satisfied: pydantic!=1.8,!=1.8.1,!=2.0.0,!=2.0.1,<3.0.0,>=1.7.4 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (2.1.1)
Requirement already satisfied: pydub in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (0.25.1)
Requirement already satisfied: python-multipart in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (0.0.6)
Requirement already satisfied: semantic-version~=2.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (2.10.0)
Requirement already satisfied: uvicorn>=0.14.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (0.23.1)
Requirement already satisfied: websockets<12.0,>=10.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from gradio==3.47.1->-r requirements_versions.txt (line 14)) (11.0.3)
Requirement already satisfied: cffi>=1.9.1 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from pygit2==1.12.2->-r requirements_versions.txt (line 15)) (1.16.0)
Requirement already satisfied: attrs>=17.3.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from aiohttp==3.8.5->-r requirements_versions.txt (line 16)) (23.1.0)
Requirement already satisfied: charset-normalizer<4.0,>=2.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from aiohttp==3.8.5->-r requirements_versions.txt (line 16)) (3.2.0)
Requirement already satisfied: multidict<7.0,>=4.5 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from aiohttp==3.8.5->-r requirements_versions.txt (line 16)) (6.0.4)
Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from aiohttp==3.8.5->-r requirements_versions.txt (line 16)) (4.0.2)
Requirement already satisfied: yarl<2.0,>=1.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from aiohttp==3.8.5->-r requirements_versions.txt (line 16)) (1.9.2)
Requirement already satisfied: frozenlist>=1.1.1 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from aiohttp==3.8.5->-r requirements_versions.txt (line 16)) (1.4.0)
Requirement already satisfied: aiosignal>=1.1.2 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from aiohttp==3.8.5->-r requirements_versions.txt (line 16)) (1.3.1)
Requirement already satisfied: importlib-metadata in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from diffusers==0.23.0->-r requirements_versions.txt (line 18)) (7.0.0)
Requirement already satisfied: wcwidth>=0.2.5 in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from ftfy==6.1.1->-r requirements_versions.txt (line 19)) (0.2.12)
Requirement already satisfied: torchvision in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from clip_interrogator==0.6.0->-r requirements_versions.txt (line 20)) (0.16.2+cu118)
Requirement already satisfied: open-clip-torch in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from clip_interrogator==0.6.0->-r requirements_versions.txt (line 20)) (2.23.0)
Collecting kornia-rs>=0.1.0 (from kornia==0.7.2->-r requirements_versions.txt (line 22))
  Using cached kornia_rs-0.1.3-cp310-none-win_amd64.whl.metadata (8.9 kB)
Requirement already satisfied: jsonschema>=3.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from altair<6.0,>=4.2.0->gradio==3.47.1->-r requirements_versions.txt (line 14)) (4.18.4)
Requirement already satisfied: toolz in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from altair<6.0,>=4.2.0->gradio==3.47.1->-r requirements_versions.txt (line 14)) (0.12.0)
Requirement already satisfied: pycparser in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from cffi>=1.9.1->pygit2==1.12.2->-r requirements_versions.txt (line 15)) (2.21)
Requirement already satisfied: contourpy>=1.0.1 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from matplotlib~=3.0->gradio==3.47.1->-r requirements_versions.txt (line 14)) (1.1.0)
Requirement already satisfied: cycler>=0.10 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from matplotlib~=3.0->gradio==3.47.1->-r requirements_versions.txt (line 14)) (0.11.0)
Requirement already satisfied: fonttools>=4.22.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from matplotlib~=3.0->gradio==3.47.1->-r requirements_versions.txt (line 14)) (4.41.1)
Requirement already satisfied: kiwisolver>=1.0.1 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from matplotlib~=3.0->gradio==3.47.1->-r requirements_versions.txt (line 14)) (1.4.4)
Requirement already satisfied: pyparsing<3.1,>=2.3.1 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from matplotlib~=3.0->gradio==3.47.1->-r requirements_versions.txt (line 14)) (3.0.9)
Requirement already satisfied: python-dateutil>=2.7 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from matplotlib~=3.0->gradio==3.47.1->-r requirements_versions.txt (line 14)) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from pandas<3.0,>=1.0->gradio==3.47.1->-r requirements_versions.txt (line 14)) (2023.3)
Requirement already satisfied: annotated-types>=0.4.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from pydantic!=1.8,!=1.8.1,!=2.0.0,!=2.0.1,<3.0.0,>=1.7.4->gradio==3.47.1->-r requirements_versions.txt (line 14)) (0.5.0)
Requirement already satisfied: pydantic-core==2.4.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from pydantic!=1.8,!=1.8.1,!=2.0.0,!=2.0.1,<3.0.0,>=1.7.4->gradio==3.47.1->-r requirements_versions.txt (line 14)) (2.4.0)
Requirement already satisfied: idna<4,>=2.5 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from requests->transformers==4.30.2->-r requirements_versions.txt (line 3)) (3.4)
Requirement already satisfied: urllib3<3,>=1.21.1 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from requests->transformers==4.30.2->-r requirements_versions.txt (line 3)) (2.0.7)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from requests->transformers==4.30.2->-r requirements_versions.txt (line 3)) (2023.7.22)
Requirement already satisfied: sympy in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from torch>=1.6.0->torchsde==0.2.5->-r requirements_versions.txt (line 1)) (1.12)
Requirement already satisfied: networkx in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from torch>=1.6.0->torchsde==0.2.5->-r requirements_versions.txt (line 1)) (2.8.8)
Requirement already satisfied: click>=7.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from uvicorn>=0.14.0->gradio==3.47.1->-r requirements_versions.txt (line 14)) (8.1.6)
Requirement already satisfied: h11>=0.8 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from uvicorn>=0.14.0->gradio==3.47.1->-r requirements_versions.txt (line 14)) (0.14.0)
Requirement already satisfied: starlette<0.28.0,>=0.27.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from fastapi->gradio==3.47.1->-r requirements_versions.txt (line 14)) (0.27.0)
Requirement already satisfied: httpcore<0.18.0,>=0.15.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from httpx->gradio==3.47.1->-r requirements_versions.txt (line 14)) (0.17.3)
Requirement already satisfied: sniffio in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from httpx->gradio==3.47.1->-r requirements_versions.txt (line 14)) (1.3.0)
Requirement already satisfied: zipp>=0.5 in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from importlib-metadata->diffusers==0.23.0->-r requirements_versions.txt (line 18)) (3.17.0)
Requirement already satisfied: sentencepiece in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from open-clip-torch->clip_interrogator==0.6.0->-r requirements_versions.txt (line 20)) (0.1.99)
Requirement already satisfied: protobuf in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from open-clip-torch->clip_interrogator==0.6.0->-r requirements_versions.txt (line 20)) (4.23.4)
Requirement already satisfied: timm in e:\diffusion\ruinedfooocus_win64_1-25-2\python_embeded\lib\site-packages (from open-clip-torch->clip_interrogator==0.6.0->-r requirements_versions.txt (line 20)) (0.9.12)
Requirement already satisfied: anyio<5.0,>=3.0 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from httpcore<0.18.0,>=0.15.0->httpx->gradio==3.47.1->-r requirements_versions.txt (line 14)) (3.7.1)
Requirement already satisfied: jsonschema-specifications>=2023.03.6 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from jsonschema>=3.0->altair<6.0,>=4.2.0->gradio==3.47.1->-r requirements_versions.txt (line 14)) (2023.7.1)
Requirement already satisfied: referencing>=0.28.4 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from jsonschema>=3.0->altair<6.0,>=4.2.0->gradio==3.47.1->-r requirements_versions.txt (line 14)) (0.30.0)
Requirement already satisfied: rpds-py>=0.7.1 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from jsonschema>=3.0->altair<6.0,>=4.2.0->gradio==3.47.1->-r requirements_versions.txt (line 14)) (0.9.2)
Requirement already satisfied: six>=1.5 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from python-dateutil>=2.7->matplotlib~=3.0->gradio==3.47.1->-r requirements_versions.txt (line 14)) (1.16.0)
Requirement already satisfied: mpmath>=0.19 in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from sympy->torch>=1.6.0->torchsde==0.2.5->-r requirements_versions.txt (line 1)) (1.3.0)
Requirement already satisfied: exceptiongroup in c:\users\mirek190\appdata\roaming\python\python310\site-packages (from anyio<5.0,>=3.0->httpcore<0.18.0,>=0.15.0->httpx->gradio==3.47.1->-r requirements_versions.txt (line 14)) (1.1.2)
Using cached einops-0.4.1-py3-none-any.whl (28 kB)
Using cached transformers-4.30.2-py3-none-any.whl (7.2 MB)
Using cached safetensors-0.3.1-cp310-cp310-win_amd64.whl (263 kB)
Using cached PyYAML-6.0-cp310-cp310-win_amd64.whl (151 kB)
Using cached Pillow-9.2.0-cp310-cp310-win_amd64.whl (3.3 MB)
Using cached scipy-1.9.3-cp310-cp310-win_amd64.whl (40.1 MB)
Using cached tqdm-4.64.1-py2.py3-none-any.whl (78 kB)
Using cached psutil-5.9.5-cp36-abi3-win_amd64.whl (255 kB)
Using cached numpy-1.23.5-cp310-cp310-win_amd64.whl (14.6 MB)
Using cached omegaconf-2.2.3-py3-none-any.whl (79 kB)
Using cached gradio-3.47.1-py3-none-any.whl (20.3 MB)
Using cached opencv_python-4.8.1.78-cp37-abi3-win_amd64.whl (38.1 MB)
Using cached kornia-0.7.2-py2.py3-none-any.whl (825 kB)
Using cached gradio_client-0.6.0-py3-none-any.whl (298 kB)
Using cached kornia_rs-0.1.3-cp310-none-win_amd64.whl (1.2 MB)
Using cached tokenizers-0.13.3-cp310-cp310-win_amd64.whl (3.5 MB)
Installing collected packages: tokenizers, safetensors, einops, antlr4-python3-runtime, tqdm, pyyaml, psutil, Pillow, numpy, kornia-rs, scipy, opencv-python, omegaconf, transformers, kornia, gradio-client, gradio
  Attempting uninstall: tokenizers
    Found existing installation: tokenizers 0.14.1
    Uninstalling tokenizers-0.14.1:
      Successfully uninstalled tokenizers-0.14.1
  Rolling back uninstall of tokenizers
  Moving to c:\users\mirek190\appdata\roaming\python\python310\site-packages\tokenizers-0.14.1.dist-info\
   from C:\Users\mirek190\AppData\Roaming\Python\Python310\site-packages\~okenizers-0.14.1.dist-info
  Moving to c:\users\mirek190\appdata\roaming\python\python310\site-packages\tokenizers\
   from C:\Users\mirek190\AppData\Roaming\Python\Python310\site-packages\~okenizers

stderr: DEPRECATION: torchsde 0.2.5 has a non-standard dependency specifier numpy>=1.19.*; python_version >= "3.7". pip 24.1 will enforce this behaviour change. A possible replacement is to upgrade to a newer version of torchsde or contact the author to suggest that they release a version with a conforming dependency specifiers. Discussion can be found at https://github.com/pypa/pip/issues/12063
ERROR: Could not install packages due to an OSError: [WinError 5] Access is denied: 'E:\\DIFFUSION\\RuinedFooocus_win64_1-25-2\\python_embeded\\Lib\\site-packages\\tokenizers\\tokenizers.cp310-win_amd64.pyd'
Consider using the `--user` option or check the permissions.

E:\DIFFUSION\RuinedFooocus_win64_1-25-2>pause
Press any key to continue . . .

**Comment by runew0lf** [comment]
You dont need to update, just run the program , it'll automatically update

**Comment by mirek190** [comment]
yes I just run and automatic update started .... so getting error and can't run the program.

**Comment by runew0lf** [comment]
looks like somethings messed up my friend, it would be easier if you joined the discord to diagnose than a 15 hour back and forth

**Comment by Shardz777** [comment]
Everything was working great for me for v1.35, then I started getting errors after the updates that that. After the last update it would "time out" upon launch and I'd have to launch it a few times for it to run. Now with v1.38.1 it won't run at all and I'm getting errors. I haven't even touched this install and the update blew it up. I now get these errors upon running, and I have no clue what the problem is as I haven't even touched it.

https://i.ibb.co/6YwvSkd/Screenshot-2024-04-23-093829.png

**Comment by runew0lf** [comment]
looks like your default settings are messed up, go into the settings folder and delete `settings.json` and it should recreate it for you mate

**Comment by Shardz777** [comment]
> looks like your default settings are messed up, go into the settings folder and delete `settings.json` and it should recreate it for you mate. You can also join our discord for quicker access to help too <3

Thanks for the fast reply! I performed a PC restart, copied over the default settings file from my backup (smart move on my part), and now it works. However, it still 'times out' while I'm entering my prompts, models, loras, etc., though after I restart it a couple times, it starts working. It's been doing this since v1.38 for me, and not just working immediately like it used to in v1.35.

Also, I can't run Fooocus due to some NVIDIA driver issue, which is why I chose RuinedFoocus since it just worked out of the box. I'm on a laptop and don't want to change my current manufacturer's driver since everything else works fine. And thank you for this amazing fork, I need to say that right now. Your efforts made it possible to be able to do this at all. I will see if I can jump on Discord later after some testing, but at least I can get it running now!  :D
